tests/functional/encryption/test_encryption_keyrotation.py

```
@@ -198,28 +224,32 @@ def test_noobaa_keyrotation(self, minutes=3):
             noobaa_keyrotation.get_noobaa_keyrotation_schedule() == schedule
         ), f"Keyrotation schedule is not set to every {minutes} minutes in Noobaa object."
 
+        logger.test_step("Compare old NooBaa keys with new keys after rotation")
         try:
             compare_noobaa_old_keys_with_new_keys(
                 noobaa_keyrotation, old_noobaa_backend_key, old_noobaa_volume_key
             )
         except UnexpectedBehaviour:
-            log.info("Noobaa Key Rotation is not happend.")
+            logger.warning("Noobaa Key Rotation did not happen.")
             assert False
```

Ruff (0.15.15)

[warning] Assertion always fails, replace with `pytest.fail()`

(PT015)